src/components/SUE/SueStatus.tsx

```diff
@@ -71,6 +90,12 @@ const styles = StyleSheet.create({
     paddingHorizontal: normalize(14),
     paddingVertical: normalize(4)
   },
+  filterContainer: {
+    borderRadius: normalize(8),
+    borderWidth: normalize(1),
+    height: normalize(32),
+    width: device.width * 0.3
```

Comment:
why * 0.3 here?

Reply:
and how does it look like in landscape

Reply:
It looks like this. It's not perfect, but I think it's ideal for the test version.
![image](https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/6f41cd99-b325-4d8c-8c2e-03e3b09a4857)